docs/website/assets/css/main.css

```diff
@@ -47,6 +47,13 @@ pre {
   overflow-x: auto;
 }
 
+code {
+  background: #f4f5f6;
+  border-style: none;
+  border-width: 0.5px;
+  border-radius: 5px;
+}
+
```

Comment:
I noticed that the docs `<code>` elements were hard to decipher from regular text.